src/main/java/org/opentripplanner/standalone/config/routerequest/RouteRequestConfig.java

```diff
@@ -241,14 +207,14 @@ cost function. The cost function (`unpreferredCost`) is defined as a linear func
 
     // Map preferences
     request.withPreferences(preferences -> mapPreferences(c, preferences));
+    request.withPreferences(preferences -> setVehicleRental(c, request, preferences));
```

Comment:
I doesn't make a difference to the execution but I think this would fit better somewhere inside the `mapPreferences` method.

Reply:
Have implemented the change